examples-generated/mq/broker.yaml

```diff
@@ -18,8 +18,8 @@ spec:
     engineVersion: 5.15.9
     hostInstanceType: mq.t2.micro
     region: us-west-1
-    securityGroups:
-    - ${aws_security_group.test.id}
+    securityGroupRefs:
+    - name: test
```

Comment:
Could you please add this parameter and related resources to `examples/mq/broker.yaml` and test it?

Reply:
Done.
Also added testhooks for this resource.
Also manually tested case when we need to use next selector:
`securityGroupSelector:
  matchLabels:
    testing.upbound.io/example-name: example`